lib/msal-node/test/TestConstants.ts

```diff
@@ -0,0 +1 @@
+export const TESTS = 100;
```

Comment:
This probably doesn't need to be constant, can put this in test file because the number or times you want to validate things may change depending on the test.  Also the naming could be clearer

Reply:
yeah, agree. Will move.

Reply:
Done